src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -527,7 +527,9 @@ public void jobFinished () {
         deployment.deployJobSummaries.add(0, new DeploySummary(this));
         Persistence.deployments.replace(deployment.id, deployment);
         // Send notification to those subscribed to updates for the deployment.
-        NotifyUsersForSubscriptionJob.createNotification("deployment-updated", deployment.id, message);
+        NotifyUsersForSubscriptionJob notifyUsersForSubscriptionJob
+            = new NotifyUsersForSubscriptionJob("deployment-updated", deployment.id, message);
+        notifyUsersForSubscriptionJob.run();
```

Comment:
Please revert due to reasons list in https://github.com/ibi-group/datatools-server/pull/348#discussion_r512341045.